operators/tokenizer/tokenizers.cc

```diff
@@ -4,18 +4,18 @@
 #include "ocos.h"
 
 #ifdef ENABLE_GPT2_TOKENIZER
-#include "bpe_tokenizer.hpp"
 #include "bpe_kernels.h"
+#include "bpe_tokenizer_model.hpp"
 #include "bpe_decoder.hpp"
 #endif
 
 #ifdef ENABLE_SPM_TOKENIZER
-#include "sentencepiece_tokenizer.hpp"
+#include "sentencepiece_tokenizer.h"
```

Comment:
needs to be changed in [operators/tokenizer/sentencepiece_tokenizer.cc](https://github.com/microsoft/onnxruntime-extensions/blob/6b94f4d7a597f0199f3e44d020d601d408605736/operators/tokenizer/sentencepiece_tokenizer.cc#L7) as well.